javascript-data-model/order-history.js

```diff
@@ -0,0 +1,79 @@
+var orderHistory = [
+  {
+    orderNumber: '113-3941689-8772232',
+    dateOrderPlaced: 'August 4, 2020',
+    totalBeforeTax: '$31.55',
+    totalAfterTax: '$34.00',
+    grandTotal: '$34.00',
+    dateDelivered: 'Aug 8, 2020',
+    shipTo: 'JS Masher',
+    returnWindowCloseDate: 'September 7th, 2020',
+    returnWindowClosed: true,
+    packageDelivered: true,
+    handedToResident: true,
+    productCategory: 'books',
+    title: 'Javascript for Impatient programmers',
+    author: 'Rauschmayer, Dr. Axel'
+  },
```

Comment:
You have data about the order mixed in the same object with data about the item purchased. Your order object should have some sort of property containing data about the "products" being purchased, you don't want to intermix all of that in one big object, there needs to be layers to this to divide things up.